app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
```

Comment:
You should be albe to pass hidden attribute when creating an Animal instance. You should make it similar to health, just use a default value